### DISCUSSION TITLE
MA Installation

### DISCUSSION THREAD

**Discussion by AlanEisen** [music-assistant/support#3123] [Q&A] [answered]
I'm running HA Green. I was able to download and install Music Assistant via HACS (and then rebooted HA). I then went to Integrations, found Music Assistant, and tried to install it. It prompted me for the local server. The default was "http://mass.local:8095" which tried to work but resulted in "Failed to Connect." I tried changing the default to my IP address:8095, but it says "Unexpected Error." I'm not sure what I'm doing wrong here, but I can't get the integration to install. Thanks for your assistance.

**Comment by OzGav** [comment, marked as answer]
https://music-assistant.io/#architecture
